be/src/service/doris_main.cpp

```diff
@@ -282,6 +282,10 @@ int main(int argc, char** argv) {
         fprintf(stderr, "you need set DORIS_HOME environment variable.\n");
         exit(-1);
     }
+    if (getenv("PID_DIR") == nullptr) {
```

Comment:
why not set a default PID_DIR?

Reply:
Sure, is it good to set it as the current dir like the `start_be` script?

Reply:
I'm not sure if it's good to set the default value here since the start_be also set a default value.